### DISCUSSION TITLE
New Icon for the manager

### DISCUSSION THREAD

**Discussion by yujincheng08** [LSPosed/LSPosed#612] [Ideas]
We are raising a new icon for the manager.

Requirements:
~~1. Contains `LSP`~~
2. Not too complex: simpler is better
3. SVG format

Submit yours in PNG format on this discussion with SVG source code. We will adopt the one in favour of maintainers and with the highest votes.

**Comment by ansarimohammed422** [comment]
Hii i have created an icon for lsposed 
here is the link:-
https://github.com/ansarimohammed422/Icons/releases/tag/new










![Lsposed Logo](https://user-images.githubusercontent.com/78704469/131149282-2bb2c8a7-7add-477b-bc6c-af4164d9ce7f.png)










to source code
consider seeing thit
